src/include/daos/drpc_modules.h

```diff
@@ -24,6 +24,8 @@
 #ifndef __DAOS_DRPC_MODULES_H__
 #define __DAOS_DRPC_MODULES_H__
 
+#define DRPC_PROTOCOL_VERSION 0x00000001
```

Comment:
It is not an overall dRPC protocol version.  It's really a GetAttachInfo definition version.  That said, Kris requested the change in the name and location where it was defined.  She seemed to feel strong about it, and I did not, so I make the change.  You can look at the discussion history if that interests you.

Reply:
Based on the ticket it sounded like the intention was to version our internal communications moving forward, which might include more than GetAttachInfo in the future, though I don't fully understand the specific issue that led to the ticket.

If a less rigid way of dealing with it, like complaining about empty fields, is an option, I'd prefer that rather than relying on version numbering.